project/task1/detectFixations.py

```diff
@@ -0,0 +1,29 @@
+import statistics
+from pointClass import Point
+from handleWindowPoints import handleWindowPoints
+
+def detectFixations(rowData, DT, SC):
+    fixationDuration = []
+    fixationCentroid = []
+    i=0
+    while i+SC-1 <= len(rowData)-1:
+        D = handleWindowPoints(rowData[i:i+SC])[0]
+        compareDispersion = D > DT
+        if compareDispersion:
+            i=i+1
+        else: 
+            j=1
+            while D<=DT and i+SC-1+j<=len(rowData)-1:
+                output = handleWindowPoints(rowData[i:i+SC+j])
+                D = output[0]
+                j=j+1
+
+            fixationDuration.append( round((SC-1+j)*0.001, 3) ) 
```

Comment:
magic numbers

Reply:
This is the number of points in the moving window (that contain fixation) * 1ms.

Reply:
yeah, 'magic number' issue means that you should create a constant to explain that number. For example here it should be `SAMPLING_INTERVAL`